ui/lib/pki/addon/routes/issuers/issuer/cross-sign.js

```diff
@@ -3,16 +3,28 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
-import PkiIssuerRoute from '../issuer';
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
 import { withConfirmLeave } from 'core/decorators/confirm-leave';
 
 @withConfirmLeave()
-export default class PkiIssuerCrossSignRoute extends PkiIssuerRoute {
+export default class PkiIssuerCrossSignRoute extends Route {
+  @service store;
+  @service secretMountPath;
+
+  model() {
+    return this.modelFor('issuers.issuer');
+  }
+
   setupController(controller, resolvedModel) {
     super.setupController(controller, resolvedModel);
-    controller.breadcrumbs.push(
+
+    controller.breadcrumbs = [
```

Comment:
I had a question for my understanding. Previously we were pushing in the two breadcrumbs, but the new change is setting this to an array of breadcrumbs. Why are we doing that?

Reply:
Good question! Before, since we were extending the other route, it was first calling `setupController` on the extended route, which instantiated `breadcrumbs` on the controller. Then this route's `setupController` is called, which just pushes the new breadcrumbs to the parent's array. Hope that makes sense!